src/ConferenceItem.jsx

```diff
@@ -1,20 +1,40 @@
 import React from "react";
-import moment from "moment";
+import styled from "styled-components";
+
+const StyledCard = styled.div`
+  width: 100%;
+  height: 200px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  background: papayawhip;
+
+  :hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  }
+`;
+
+const StyledListItem = styled.li`
+  width: 300px;
+  margin: 24px;
+`;
 
 const ConferenceItem = ({ conference }) => {
   const { name, topic, dateTime, city } = conference;
-  const date = moment(dateTime).format("DD/MM/YYYY");
-  const time = moment(dateTime).format("HH:mm");
+  const date = new Date(dateTime);
```

Comment:
I'd suggest for readability creating day, month, year up here and using the variables below in the render. ie 
`const month = date.getMonth();` and so on. It'll make the `p` tag clearer

Reply:
Agreed, we implemented this in the latest commit, definitely looks nicer!